src/System.Diagnostics.Process/ref/System.Diagnostics.Process.cs

```diff
@@ -23,75 +22,74 @@ public partial class DataReceivedEventArgs : System.EventArgs
         public string Data { get { throw null; } }
     }
     public delegate void DataReceivedEventHandler(object sender, System.Diagnostics.DataReceivedEventArgs e);
-    public partial class Process : System.ComponentModel.Component
+    [System.AttributeUsageAttribute(System.AttributeTargets.All)]
+    public partial class MonitoringDescriptionAttribute : System.ComponentModel.DescriptionAttribute
+    {
+        public MonitoringDescriptionAttribute(string description) { }
+        public override string Description { get { throw null; } }
+    }
+    public partial class Process : System.ComponentModel.Component, System.IDisposable
```

Comment:
Is this change fine? Adding `IDisposable`.

cc @wtgodbe, @krwq

Reply:
This should be fine, given that it didn't trigger any APICompat checks